### PR TITLE
feat(cluster-autoscaler/exoscale): add support for --nodes

### PR DESCRIPTION
**Pull request by thomasstxyz**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

**cluster-autoscaler/exoscale:** This PR adds support for configuration of minimum and maximum size of specified node groups via the `--nodes=<min>:<max>:<nodepool-name>` flag.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by Sapd** [comment]
FYI @pierre-emmanuelJ @7fELF @PhilippeChepy

**Review by pierre-emmanuelJ — APPROVED** [review]
Thanks a lot for this change! Nice work, it's clear :)

**Review by sauterp — APPROVED** [review]

**Comment by thomasstxyz** [comment]
Please review and approve @gjtempleton @feiskyer @BigDarkClown thanks

File: charts/cluster-autoscaler/Chart.yaml
Comment (Shubham82) [inline charts/cluster-autoscaler/Chart.yaml:14]: @thomasstxyz
This is a new feature, please make this a minor version bump.

Comment (thomasstxyz) [reply charts/cluster-autoscaler/Chart.yaml]: @Shubham82 Done.

**Comment by Shubham82** [comment]
Thanks @thomasstxyz
/lgtm

**Comment by thomasstxyz** [comment]
@Shubham82 is there anything to do from my side?

**Comment by Shubham82** [comment]
> @Shubham82 is there anything to do from my side?

For me, it looks good.
To merge it, PR needs to be approved by the approver (`/approve` label).

**Comment by Shubham82** [comment]
CC @gjtempleton could you please approve this PR so it will be merged?

Thanks!

**Comment by Shubham82** [comment]
@thomasstxyz, please resolve the merge conflict.

**Comment by Shubham82** [comment]
Hi @gjtempleton @jackfrancis 
Need your approval to merge this PR.

Thanks!

**Comment by gjtempleton** [comment]
Thanks for this, good to see.
LGTM on the helm chart side.

/approve

**Comment by dev-dsp** [comment]
Is it nodepool-name? Or nodepool-id? Or instancepool-id?

**Comment by thomasstxyz** [comment]
> Is it nodepool-name? Or nodepool-id? Or instancepool-id?

nodepool-name

**Pull request merged**
